### PR TITLE
Update to v1.3.0

### PR DESCRIPTION
**Pull request by srl295**
Fixes: https://github.com/IBM-Bluemix/gp-nodejs-sample/issues/8

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ var mybundle = gpClient.bundle(bundleName);
 // Example: /
 app.get('/', function (req, res) {
   // For the root page, load the list of supported langs 
-  mybundle.getInfo({}, function (err, projInfo) {
+  mybundle.getInfo(function (err, projInfo) {
```
Comment (srl295) [inline app.js:38]: in v1.3.0, where  the `{}` options are ignored or optional, they may be omitted.

File: app.js [hunk 2]
```diff
@@ -49,12 +49,8 @@ app.get('/', function (req, res) {
       res.write('<h1>Welcome to the Globalization Pipeline Node.js sample!</h1>');
       res.write('<h2> Pick a language code:</h2>');
       res.write('<ul>');
-      // add the source (probably english)
-      addLang(projInfo.sourceLanguage);
-      // add all of the targets
-      if( projInfo.targetLanguages) {
-        projInfo.targetLanguages.forEach( addLang );
-      }
+      projInfo.languages() // --> Array: [ 'en', 'de', … ]
```
Comment (srl295) [inline app.js:52]: * `projInfo` is now a `Bundle` object here
* the new `languages()` function returns both the source AND target languages, in a single convenient array!

File: app.js [hunk 3]
```diff
@@ -65,12 +61,12 @@ app.get('/', function (req, res) {
 app.get(/^\/(\w+)\/hello/, function (req, res) {
   var lang = req.params[0];
   // Just get this string
-  mybundle.getEntryInfo({ resourceKey: 'hello', languageId: lang}, function (err, data) {
+  mybundle.entry({ resourceKey: 'hello', languageId: lang})
+   .getInfo(function (err, entry) {
```
Comment (srl295) [inline app.js:65]: the `ResourceEntry` object supports `getInfo()` which returns another `ResourceEntry` object, but populated with the data

File: package.json [hunk 4]
```diff
@@ -12,7 +12,7 @@
   "dependencies": {
     "cfenv": "^1.0.3",
     "express": "^4.14.0",
-    "g11n-pipeline": "^1.2.0",
+    "g11n-pipeline": "^1.3.0",
```
Comment (srl295) [inline package.json:15]: All you need to do… is `1.3.0` - and you're through!

File: app.js [hunk 5]
```diff
@@ -4,7 +4,7 @@
  */
  
 /*	
- * Copyright IBM Corp. 2015
+ * Copyright IBM Corp. 2015,2016
```
Comment (srl295) [inline app.js:7]: OK, it's been a while.

File: app.js (hunk 2 above)
Comment (steveatkin) [reply app.js:52]: This seems like a good change.

File: app.js (hunk 3 above)
Comment (steveatkin) [reply app.js:65]: I am not sure I understand. Do you mean it will have the target translation data and the source as well?

**Review by steveatkin — APPROVED** [review]
On whole it looks good to me. Take a look at my comments.

**Comment by steveatkin** [comment]
Good to go.

File: app.js (hunk 3 above)
Comment (srl295) [reply app.js:65]: >  mybundle.entry({ resourceKey: 'hello', languageId: lang})
>   .getInfo(function (err, theEntry) { …} )

`theEntry` contains the translation data for languageId `lang`

Comment (srl295) [reply app.js:65]: Or put another way:
> var anEntry = mybundle.entry({ resourceKey: 'hello', languageId: lang});

`anEntry` is now a `ResourceEntry` _object_  It does not have any translation data in it. But you can call `.getInfo`…

> anEntry.getInfo(function (err, entryWithInfo) { … } );

… Now, `entryWithInfo` has the full information for the entry - the `value`, the `updatedAt` (mod time), etc.  Also, `entryWithInfo` is itself a `ResourceEntry` object, so  you could make further calls.

**Pull request merged**